joints:
  sting_gripper_joint:
    type: PositionControlledJoint
    default_rpm: 10.0
